modules/effects/src/main/scala/dev/profunktor/redis4cats/algebra/connection.scala

```diff
@@ -21,4 +21,6 @@ trait ConnectionCommands[F[_]] extends Ping[F]
 trait Ping[F[_]] {
   def ping: F[String]
   def select(index: Int): F[Unit]
+  def auth(password: CharSequence): F[Boolean]
```

Comment:
Just to follow along the other algebras I think it would be better to introduce a new `trait Auth[F]` instead of putting the `auth` methods in the `Ping[F]` trait.